### ISSUE TITLE
move PPI and maybe also other PPI-related repositories into the Perl-Critic organization?

### ISSUE THREAD

**Issue by wchristian** [Perl-Critic/Perl-Critic#859]
Adam has gotten back to me and confirmed he's happy to give the project over entirely. I'd prefer to move the PPI repo into an org, and the question now is:

Should i make my own org, or would you prefer to adopt it into the Perl-Critic org? (This would entail giving Adam and me admin rights.)

**Comment by wbraswell** [comment]
I am in favor.  It would make my job easier.

**Comment by wbraswell** [comment]
@petdance @thaljef 
What are your thoughts?

**Comment by petdance** [comment]
I don't see the advantage of combining the two projects under one organization, but I'm listening.  @wbraswell, how would it help make your job easier?

**Comment by wbraswell** [comment]
@petdance 
I am currently working on several issues in both PPI & Perl::Critic, some of which are related to both projects.  Having one organization would streamline the bug fix process and require fewer GitHub issues, etc.  Since Perl::Critic is based on PPI, and there is potential interest in the PPI group to be merged into the Perl::Critic group, then I think we should give their proposal due consideration.

**Comment by wchristian** [comment]
I was also thinking of giving people who're actively maintaining Perl-Critic COMAINT on PPI so emergency fixes can be put out faster, and generally wish to increase the pool of people who have an eye on PPI and can facilitate adding/moving maintainers on the repository level without needing to refer back to a single authority.

As far as PAUSE goes i'd still need to be poked, but with multiple COMAINTS available, and PAUSE admins being available to adjust things that's a concern with different parameters.

That said, i'm also happy to just roll my own org if there's no concensus on interest.

**Comment by wbraswell** [comment]
I agree with all of @wchristian 's points, and I support the proposal.

**Comment by petdance** [comment]
So it sounds like the plan is to 1) give some number of people admin rights to the Perl-Critic organization, and 2) those people would then move PPI over under the Perl-Critic organization.  Correct?

So who needs admin rights?

**Comment by wchristian** [comment]
Correct. Adamkennedy to move it and me enough rights to manage it. And I'd
need to know from you who would want comaint on it.

**Comment by wbraswell** [comment]
@petdance 
I believe that is correct.

@wchristian 
I have a permanent vested interest in PPI, because RPerl requires it.  I will continue to help with bugs and releases as needed, and will be happy to hold comaint as a backup person.

**Comment by petdance** [comment]
I have invited @wchristian to be an admin in Perl-Critic.  I'll let him take it from there, coordinating with Adam about the transfer.

I don't think that PAUSE rights need to change anywhere so long as there's coordination between PPI & P::C releases.

**Comment by wchristian** [comment]
@petdance Thanks for the trust. I've accepted, invited and informed Adam on what he'd need to do, and hope to receive favourable news from him.

And as soon as the pause admins respond, i'll provide @wbraswell with co-maint. :)

**Comment by petdance** [comment]
Thanks for driving it all so I don't have to think about it. :-)

**Comment by petdance** [comment]
Is there anything more to do here, @wchristian ?
